R/filters.R

```diff
@@ -0,0 +1,46 @@
+manage_filters <- function(filters) {
```

Comment:
Would you be able to add documentation comments for this function?

Reply:
Here `manage_filters` handles filters to be passed to `get_orthogonal_selection` https://github.com/keller-mark/pizzarr/commit/cbaeca6bf8f6ff928682a9c06d9b7d0c5a493066